spec/lib/hubspot/contact_list_spec.rb

```diff
@@ -159,21 +194,25 @@
 
       context 'when the contact list is not found' do
         it 'raises an error' do
-          expect { Hubspot::ContactList.find(-1) }.to raise_error(Hubspot::RequestError)
+          expect { Hubspot::ContactList.find(-1) }.to raise_error(Hubspot::NotFoundError)
         end
       end
     end
 
     context 'given a list of ids' do
       cassette "contact_list_batch_find"
 
+      let(:list1) { Hubspot::ContactList.create!(name: SecureRandom.hex) }
+      let(:list2) { Hubspot::ContactList.create!(name: SecureRandom.hex) }
+      let(:list3) { Hubspot::ContactList.create!(name: SecureRandom.hex) }
+
       it 'find lists of contacts' do
-        lists = Hubspot::ContactList.find([2,3,4])
+        lists = Hubspot::ContactList.find([list1.id,list2.id,list3.id])
         list = lists.first
         expect(list).to be_a(Hubspot::ContactList)
-        expect(list.id).to be == 2
-        expect(lists.second.id).to be == 3
-        expect(lists.last.id).to be == 4
+        expect(list.id).to be == list1.id
+        expect(lists.second.id).to be == list2.id
+        expect(lists.last.id).to be == list3.id
```

Comment:
un risque qu'un jour ce ne soit pas le même ordre ?
```suggestion
        expect(lists.map(&:id)).to contain_exactly(list1.id, list2.id, list3.id)
```